src/main/java/io/fabric8/maven/docker/access/chunked/PullOrPushResponseJsonHandler.java

```diff
@@ -30,23 +30,23 @@ public void process(JSONObject json) throws DockerAccessException {
     private void logInfoMessage(JSONObject json) {
         String value;
         if (json.has("stream")) {
-            value = json.getString("stream").replaceFirst("\n$", "");
+            value = json.optString("stream").replaceFirst("\n$", "");
```

Comment:
@rhuss just as a note, I choose `optString` as it doesn't toss a checked exception, instead it returns a sensible/ aka default empty, value.